src/Sheet.php

```diff
@@ -133,8 +133,10 @@ public function fromView(FromView $sheetExport)
 
         /** @var Html $reader */
         $reader = IOFactory::createReader('Html');
-        $reader->setSheetIndex($spreadsheet->getActiveSheetIndex());
+        // dump($spreadsheet->getActiveSheetIndex()); // Implementation bug: always return 0
+        $reader->setSheetIndex($spreadsheet->getSheetCount() - 1); // Insert content into the last sheet
         $reader->loadIntoExisting($tempFile, $spreadsheet);
+        $reader->setSheetIndex(0); // return to the first sheet as active
```

Comment:
Same here about the comment :)

Reply:
I didn't see that this actually did anything in use. Removed but there could be a case in the future where this is needed. The intention was that the generated file would focus the first sheet when opened but it did not.